### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by cfpwastaken**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/lyrics-error.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: Lyrics error
+about: Report a mistake in the lyrics we have.
+title: "[LYRICS] "
+labels: lyrics
```
Comment (j0code) [inline .github/ISSUE_TEMPLATE/lyrics-error.md:5]:
```suggestion
labels: lyrics mistake
```

**Review by j0code — CHANGES_REQUESTED** [review]
very gud templates

File: .github/ISSUE_TEMPLATE/lyrics-error.md (hunk 1 above)
Comment (cfpwastaken) [reply .github/ISSUE_TEMPLATE/lyrics-error.md:5]: ???? there is only the [lyrics] tag/label. you made it

Comment (j0code) [reply .github/ISSUE_TEMPLATE/lyrics-error.md:5]: I added the label before suggesting

Comment (cfpwastaken) [reply .github/ISSUE_TEMPLATE/lyrics-error.md:5]: image

**Review by j0code — APPROVED** [review]
seal of approval

**Pull request merged**